packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts

```diff
@@ -205,6 +216,15 @@ describe('when the application protocol returns an action', () => {
     const finalState = {...state, isFinal: true, turnNum: turnNum + 1};
     const p = wallet.pushMessage({
       signedStates: [serializeState(stateSignedBy([bob()])(finalState))],
+      objectives: [
+        {
+          type: 'CloseChannel',
+          participants: [],
```

Comment:
For some reason `participants` is required in `Objective` but I don't think it needs to be

Reply:
That field is used in the xstate wallet, so removing it from `wire-format` would be breaking.